src/main/java/ru/itis/controllers/WishListController.java

```diff
@@ -35,16 +36,17 @@ public WishList getWishList(@PathVariable("list-id") Long listId, @RequestHeader
     @PostMapping("/lists/{list-id}")
     @PreAuthorize("isAuthenticated()")
     @ApiOperation("Add new item to wish list")
-    public void addNewItem(@PathVariable("list-id") Long listId, ItemForm itemForm, @RequestHeader(name = "AUTH") String token) {
-        itemService.addNewItem(itemForm, listId);
+    public void addNewItem(@PathVariable("list-id") Long listId, ItemDto itemDto, @RequestHeader(name = "Authorization")
```

Comment:
Parameter list may be too long.

Reply:
fixed

Reply:
No, there are still 3 parameters.

Reply:
if I remove Authorozation header, user won't be able to authorize

Reply:
maybe it's time to switch to jwt tokens?